### PR TITLE
50005_review-destroy

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,5 +1,16 @@
 class ReviewsController < ApplicationController
+
```
Comment (abdullahallmehedi) [inline app/controllers/reviews_controller.rb]: remove

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+  def destroy
+    @review = Review.find params[:id]
+    if @review.destroy
+      respond_to do |format|
+        format.js
+      end
+    end
+  end
+
```
Comment (abdullahallmehedi) [inline app/controllers/reviews_controller.rb]: remove

File: app/models/ability.rb [hunk 3]
```diff
@@ -16,6 +16,7 @@ def initialize user
       can :manage, Relationship do |relationship|
         relationships.include? relationship
       end
+      can :destroy, [Review]
```
Comment (abdullahallmehedi) [inline app/models/ability.rb]: remove

File: app/views/books/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h1> <%= t ".review" %> </h1>
+    </div>
+    <% @reviews.each do |review| %>
+      <div class="panel-body">
+        <div><b><%= t ".content" %> : </b> <%= review.content %></div>
+        <div><b><%= t ".rating" %> : </b> <%= review.rating %></div>
+        <% if can?(:destroy, review) %>
+          <%= link_to "Delete", review, remote: true, method: :delete %>
```
Comment (abdullahallmehedi) [inline app/views/books/show.html.erb]: i18n

**Comment by Rownaknira** [comment]
ok

**Comment by SSalekin** [comment]
ok

File: app/controllers/reviews_controller.rb [hunk 5]
```diff
@@ -2,4 +2,11 @@ class ReviewsController < ApplicationController
   def index
     @activities = Review.review_acitvities
   end
+
+  def destroy
+    @review = Review.find params[:id]
```
Comment (dieunb) [inline app/controllers/reviews_controller.rb]: cancancan???

**Pull request merged**